src/Faker/Provider/ar_SA/Utils.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Faker\Provider\ar_SA;
+
+class Utils
+{
+    // ripped unashamedly from https://github.com/grahamking/darkcoding-credit-card/blob/master/gencc.php
+    public static function generateLuhnNumber($prefix, $length)
```

Comment:
You already have everything in `Faker\Calculator\Luhn`. No need to reimplement it here.